src/views/Navbar.tsx

```diff
@@ -0,0 +1,31 @@
+import { BsPersonCircle } from 'react-icons/bs';
+import { useLocation } from 'react-router-dom';
+import { CustomLink } from '../components/NavbarLink';
+import '../styles/navbar/navbar.css'
+
+const onLogoutClick = () => {
+  console.log('Kirjaudu ulos clicked.');
```

Comment:
Vähän häiritsee finglish. Käytetään koodissa englantia

```suggestion
  console.log('Log out button clicked');
```